suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
Although I think we have finished the discussion, I'm not completely sure I am with you on your last statement.

There are some many things that could 'in principle' be handled by software, for example indentation in languages that use `{` and `}` for code blocks or whitespace at the end of the line.

_English is not my native language, the stuff below is in a happy tone without any form of grief (typed text can be tricky at times)_
In any case, if I encounter a piece of comment that I think should be formatted differently I will just change it :-) I'm ok with you using any style you prefer, and if you do not care about single vs multiline I wont start a discussion over it in pull requests.